db/migrate/20201004014328_add_nonprofit_id_to_campaigns.rb

```diff
@@ -0,0 +1,5 @@
+class AddNonprofitIdToCampaigns < ActiveRecord::Migration[6.0]
+  def change
+    add_column :campaigns, :nonprofit_id, :integer
```

Comment:
This should also have a foreign key and index

Reply:
add_reference : campaigns, :nonprofit, index: true

Reply:
Did not know about reference column! Thanks for sharing